public/locales/es/translations.json

```
@@ -0,0 +1,49 @@
+{
+	"root.description": "Una alternativa al ecosistema de Nix",
+
+	"header.community": "Comunidad",
+
+	"goals.title": "Objetivos",
+	"goals.independent.title": "Independiente",
+	"goals.independent": "Nosotros forkearemos y mantendremos Nix, NixPkgs y NixOS. Soporte adicional para proyectos como Nix Darwin y Home Manager será considerado.",
+	"goals.gov.title": "Gobierno",
+	"goals.gov": "El proyecto se gestionará siguiendo un modelo con posiciones electas. Un comité de dirección poveerá un rumbo conjuntamente con otros comités dedicados al manejo de tareas logísticas específicas.  Características o proyectos individuales quedarán a cargo de Grupos de Interés Especial GIEs (SIG por sus siglas en Inglés). Los GIEs colaborarán en Grupos de Trabajo para cumplir sus objetivos. Cada comité y grupo tendrá sus propios objetivos, requisitos y procesos de gestión claramente definidos.",
+	"goals.stabilization.title": "Estabilización",
+	"goals.stabilization": "Funciones para facilitar la experiencia de usuario como la interfaz de comandos v3 de Nix estarán habilitados por defecto. Los Nix Flakes se usan demasiado como para eliminarlos o cambiarlos y en su lugar serán estabilizados como una v0 en sy estado actual con cualquier futuro trabajo gestionado por un Grupo de Interés Especial.",
+	"goals.infra.title": "Infraestructura",
+	"goals.infra": "Desarrollaremos y mantendremos las herramientas e infraestructura necesarias para conservar un ecosistema sano, incluyendo sistemas de integración continua y almacenamiento de binarios",
+	"goals.education.title": "Educación",
+	"goals.education": "Documentación unificada, tutoriales, guías y ejemplos darán una mejor incorporación para neofitos así como un ciclo de iteración más rápido para los expertos.",
+
+	"roadmap.title": "Hoja de ruta",
+	"roadmap.soft_fork.title": "Separación Blanda",
+	"roadmap.soft_fork.1": "Esta fase inicial contará con una estructura de dirección ad hoc debido a las circustancias de arranque. Los Valores, Objectivos y Hoja de Ruta serivrán para mantener a todos los contribuidores alineado y seguros de que su trabajo no cambiará.",
+	"roadmap.soft_fork.2": "La separación inicial de Nix y NixPkgs se dará en esta fase. Se asignran roles a los nuevos contribuidores por tal de gestionar el mantenimiento y la implementación de cambios de las fuentes. Planeamos avanzar de fase cuando el número suficiente de contribuidores se haya unído al proyecto para sotener la transición a una separación dura.",
```

```suggestion
	"roadmap.soft_fork.2": "La separación inicial de Nix y NixPkgs se dará en esta fase. Se asignarán roles a los colaboradores recién llegados para de gestionar el mantenimiento y la implementación de cambios de las fuentes. Planeamos avanzar a la siguiente fase cuando suficientes colaboradores se hayan unido al proyecto para poder sostener la transición a una separación dura.",
```

It seems that GitHub translates contributors into "colaboradores", not "contribuidores" https://docs.github.com/es/repositories/viewing-activity-and-data-for-your-repository/viewing-a-projects-contributors

---

Applied.